### PR TITLE
add recipe terraform-provider-keycloak

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute will you be a maintainer?

File: recipes/terraform-provider-keycloak/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  script:
+    - pushd {{ pkg_src }}
+    - go build -v -o $GOBIN/{{ x4_plugin }} .  # [unix]
+    - go build -v -o %GOBIN%\{{ x4_plugin }} .  # [win]
+
+requirements:
+  build:
+    - {{ compiler('go') }}
+  run:
+    - terraform ==0.11.14
```
Comment (scopatz) [inline recipes/terraform-provider-keycloak/meta.yaml]: Does this need to be so tightly pinned?

Comment (rluria14) [reply recipes/terraform-provider-keycloak/meta.yaml]: @scopatz thanks, just fixed this

Comment (rluria14) [reply recipes/terraform-provider-keycloak/meta.yaml]: @scopatz please merge when you have a chance. thanks!

**Comment by scopatz** [comment]
CC @oblute

**Comment by rluria14** [comment]
> CC @oblute

@oblute has agreed to maintain

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**